site/docs/v1/tech/lambdas/index.adoc

```diff
@@ -205,6 +206,22 @@ function populate(jwt, user, registration) {
 }
 ----
 
+=== Exceptions
+
+When an exception is thrown, further processing ceases.
+Whatever operation the lambda was part of (login, Identity Provider user reconciliation, etc) will not complete.
+
```

Comment:
Maybe a mention of enabling debug in the lambda config to troubleshoot?

Reply:
Added!